packages/gatsby-source-prismic-graphql/src/components/WrapPage.tsx

```diff
@@ -50,17 +50,17 @@ export class WrapPage extends React.PureComponent<any, WrapPageState> {
   get params() {
     const params: any = { ...this.props.pageContext };
 
-    const keys: any = [];
+    const keys: Key[] = [];
     const re = pathToRegexp(get(this.props.pageContext, 'matchPath', ''), keys);
     const match = re.exec(get(this.props, 'location.pathname', ''));
 
-    const matchFn = pathToRegexp.match(get(this.props.pageContext, 'matchPath', ''), {
+    const matchFn = matchRegex(get(this.props.pageContext, 'matchPath', ''), {
       decode: decodeURIComponent,
     });
 
     const pathParams = (() => {
       const res = matchFn(get(this.props, 'location.pathname', ''));
-      return res.params;
+      return res === false ? '' : res.params;
```

Comment:
pathParams is supposed to return an Object so your default value should not be ''

Reply:
it was me as a dumb "fix" :p

Reply:
Done